README.md

```diff
@@ -34,42 +34,74 @@ You can specify the targets and options for the task using the normal Grunt conf
 
 For edge cases, you can set some path options:
 
-### sourceFile
+#### sourceFile
 Type: `string`
 Default value: `"en.json"`
 
 The JSON file providing the primary messages.
 
-### documentationFile
+#### documentationFile
 Type: `string`
 Default value: `"qqq.json"`
 
 The JSON file providing the documentation messages.
 
-### requireMetadata
+#### requireMetadata
 Type: `boolean`
 Default value: `true`
 
 Whether to fail if message files don't have a `@metadata` meta-data key.
 
-### requireCompleteMessageDocumentation
+#### requireCompleteMessageDocumentation
 Type: `boolean`
 Default value: `true`
 
 Whether to fail if any message is in the primary file but not documented.
 
-### requireNonEmptyDocumentation
+#### disallowEmptyDocumentation
 Type: `boolean`
 Default value: `true`
 
 Whether to fail if any message is in the primary file but documented as a blank string.
 
-### requireNoUnusedDocumentation
+#### disallowUnusedDocumentation
 Type: `boolean`
 Default value: `true`
 
 Whether to fail if any documented message isn't in the primary file.
 
+#### disallowBlankTranslations
+Type: `boolean`
+Default value: `false`
+
+Whether to fail if any message is translated as a blank string.
+
+#### disallowDuplicateTranslations
+Type: `boolean`
+Default value: `false`
+
+Whether to fail if any message is translated as identical to the original string.
+
+#### disallowUnusedTranslations
+Type: `boolean`
+Default value: `false`
+
+Whether to fail if any translated message isn't in the primary file.
+
+#### requireCompleteTranslationLanguages
+Type: `string[]`
+Default value: ``
```

Comment:
This is invalid syntax. Use "View rich diff" to see how this renders. It parses the next label as code, and the next code as text.
> <img width="944" alt="capture" src="https://cloud.githubusercontent.com/assets/156867/9557741/c2a5708c-4dde-11e5-93b0-ad662e938cec.png">